doc/source/output/html.rst

```diff
@@ -59,6 +59,16 @@ but :option:`--html-details<gcovr --html-details>` and
 in order to avoid problems with the `Content Security Policy <CSP_>`_
 of some servers, especially Jenkins.
 
+The :option:`--html-template-dir<gcovr --html-template-dir>` option allows you to use an
+alternate directory with Jinja2 templates to allow you to fully customize
+the HTML output in detail.  It is advised that you start by copying and modifying
+`the existing templates <https://github.com/gcovr/gcovr/tree/master/gcovr/writer/html/templates>`_
+. Keep in mind the template context data that is passed to these templates *MAY* change
+between release versions.
+
+Note that you do not have to copy every single template and can copy and edit only the
+templates you wish to customize.
+
```

Comment:
I would rewrite this part a little bit:
```suggestion
The :option:`--html-template-dir<gcovr --html-template-dir>` option allows you to use an
alternate directory with Jinja2 templates to fully customize
the HTML output in detail.  It is advised that you start by copying the part you want to modify from
`the existing templates <https://github.com/gcovr/gcovr/tree/master/gcovr/writer/html/templates>`_
. Keep in mind the template context data that is passed to these templates *MAY* change
between release versions.

Note that you do not have to copy every single template and can copy and edit only the
templates you wish to customize.

```

Can you also change line 46 to the following content? I saw that `--html-details` is used twice in the sentence.
```
Note that the :option:`--html<gcovr --html>` and
```

Reply:
This isn't resolved yet. The first sentence doesn't make sense and the recommendation should be to only overwrite what's needed and not all templates.